packages/cli/src/config/Config.definition.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import type { ICommandDefinition } from "@zowe/imperative";
+import { SshSession } from "@zowe/zos-uss-for-zowe-sdk";
+import { Constants } from "../Constants";
+import { ConfigSetupDefinition } from "./setup/Setup.definition";
+
+const ConfigDefinition: ICommandDefinition = {
+    name: "config",
+    aliases: ["cfg"],
+    summary: "Manage Configurations",
+    description: "Manage Configurations of SSH Profiles",
+    type: "group",
+    children: [ConfigSetupDefinition],
+    passOn: [
+        {
+            property: "options",
+            value: [...SshSession.SSH_CONNECTION_OPTIONS, Constants.OPT_SERVER_PATH],
+            merge: true,
+            ignoreNodes: [{ type: "group" }],
+        },
+    ],
```

Comment:
Does the new command make use of the `--server-path` option to update the profile that is created? If not, we can probably delete the `passOn` definition.